subsys/random/CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ zephyr_library()
 zephyr_library_sources_ifdef(CONFIG_USERSPACE           rand32_handlers.c)
 endif()
 
+if (CONFIG_TEST_RANDOM_GENERATOR)
+  message(WARNING "This application is using a not truly random generator."
```

Comment:
I am not a native English speaker but to my ears it would sound better to say
`This application is not using a truly random generator.`

Reply:
yeah

Reply:
That makes sense for me as well to be honest. Just use this phrase because it was in the kconfig option. I'm going to change both then.